flexget/__init__.py

```diff
@@ -1,12 +1,16 @@
 #!/usr/bin/python
 import os
 import sys
+from pathlib import Path
 
-# __version__ import need to be first in order to avoid circular import within logger
-from ._version import __version__  # noqa
 from flexget import log
 from flexget.manager import Manager
 
+# __version__ import need to be first in order to avoid circular import within logger
```

Comment:
Oops, this got moved to not be first. Going to break things?

Reply:
fixed and added an exclude in pre-commit